packages/search-ui/src/requestAdapters.js

```diff
@@ -0,0 +1,34 @@
+function removeName(v) {
+  if (v && v.name) {
+    // eslint-disable-next-line
+    const { name, ...rest } = v;
+    return {
+      ...rest
+    };
+  }
+
+  return v;
+}
+
+function rollup(f) {
+  // TODO Move this to the app search connector and just say "app search connector"
```

Comment:
My model says that every type of filter can support nested ORs.

Meaning:
```
{
  field: "field",
  values: ["value"],
  type: "and"
}
```

```
{
  field: "field",
  values: [{from: 1, to: 100}, {from: 900, to: 1000}],
  type: "and"
}
```

Thee above would be valid in my model. However, the App Search API does not support the second case. I think it probably should, but doesn't. I will log an issue to see if we can get support for this.

Reply:
Meaning, I could envision a case where someone facets on a number of ranges, then wants to select multiple ranges within that facet to filter upon.